openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -846,6 +847,7 @@ The following claims are used in the Credential Response:
 * `transaction_id`: OPTIONAL. A JSON string identifying a Deferred Issuance transaction. This claim is contained in the response if the Credential Issuer was unable to immediately issue the credential. The value is subsequently used to obtain the respective Credential with the Deferred Credential Endpoint (see (#deferred-credential-issuance)). It MUST be present when the `credential` parameter is not returned. It MUST be invalidated after the credential for which it was meant has been obtained by the Wallet.
 * `c_nonce`: OPTIONAL. JSON string containing a nonce to be used to create a proof of possession of key material when requesting a Credential (see (#credential_request)). When received, the Wallet MUST use this nonce value for its subsequent credential requests until the Credential Issuer provides a fresh nonce.
 * `c_nonce_expires_in`: OPTIONAL. JSON integer denoting the lifetime in seconds of the `c_nonce`.
+* `callback_id`: OPTIONAL. A JSON string identifying an issued Credential that the Wallet includes in the callback as defined in (#callback).
```

Comment:
discussed during IIW that `callback_instance_id` is better suited.

Reply:
It might be worthwhile to think about whether the transaction id can be re-used. So the logic would be:

- If credential is provided and no transaction id: Immediate issuance of credential, no callback.
- If credential is provided and transaction id: Immediate issuance of credential, call callback when endpoint is supported.
- If no credential is provided but transaction id: Call deferred credential endpoint, afterwards call callback when endpoint is supported.

This avoids one more data field the client has to handle.

Reply:
```suggestion
* `callback_id`: OPTIONAL. A string identifying an issued Credential that the Wallet includes in the callback as defined in (#callback).
```

Reply:
change made